Userland/Applications/PixelPaint/Filters/Filter.cpp

```diff
@@ -40,11 +41,10 @@ void Filter::apply() const
 {
     if (!m_editor)
         return;
-    if (auto* layer = m_editor->active_layer()) {
-        apply(layer->content_bitmap(), layer->content_bitmap());
-        layer->did_modify_bitmap(layer->rect());
-        m_editor->did_complete_action(String::formatted("Filter {}", filter_name()));
-    }
+    // FIXME: I am not thread-safe!
```

Comment:
Maybe we should make the bitmap uneditable when the image processor is running?

Reply:
That's the entire reason the FIXME is here. I have something planned for that, but let's just get this merged first :^)